cwlcon2024-site/index.html

```diff
@@ -93,6 +93,13 @@ <h2>Sponsors</h2>
 
     <p> This could be your organization!! </p>
 
+    <h2>Previous Conference Links</h2>
+    <ul>
+        <li><a href="cwl.discourse.group/t/schedule-of-confirmed-talks/753">CWLCon2023</a></li>
```

Comment:
This is missing the protocol. I am not sure if it will work when rendered, did you test it locally?

Reply:
Resolved in https://github.com/common-workflow-language/cwlcon2024/pull/37/commits/1cf21abd0ef2d763a2dd5766dbbf4c88ed8cffaf